pkgs/applications/networking/zerobin/default.nix

```diff
@@ -1,10 +1,28 @@
 { lib
-, python3Packages
+, python3
 , fetchFromGitHub
 , nodePackages
 }:
 
-python3Packages.buildPythonApplication rec {
+let
+  py = python3.override {
+    packageOverrides = self: super: {
+      self = py;
+
+      bleach = super.bleach.overridePythonAttrs (oldAttrs: rec {
+        version = "3.3.1";
+
+        src = oldAttrs.src.override {
+          inherit version;
+          sha256 = "sha256-MGSDpal5VHQWCtV/zj3dG1BVHpge7Y4VpYLTTO8oqvo=";
+        };
+      });
```

Comment:
AFAICS `zerobin` is working with `bleach` 4.1.0 (the current version in Nixpkgs), so it's cleaner to just relax the upper constraint below:
```diff
diff --git a/pkgs/applications/networking/zerobin/default.nix b/pkgs/applications/networking/zerobin/default.nix
index 16a52d67045..56d5274751f 100644
--- a/pkgs/applications/networking/zerobin/default.nix
+++ b/pkgs/applications/networking/zerobin/default.nix
@@ -40,7 +40,7 @@ python3Packages.buildPythonApplication rec {
     # relax version constraints of some dependencies
     substituteInPlace setup.cfg \
       --replace "clize==4.1.1" "clize" \
-      --replace "bleach==3.1.5" "bleach>=3.1.5,<4" \
+      --replace "bleach==3.1.5" "bleach>=3.1.5,<5" \
       --replace "bottle==0.12.18" "bottle>=0.12.18,<1" \
       --replace "Paste==3.4.3" "Paste>=3.4.3,<4"
   '';
```

Reply:
Oh, I'm not sure how I missed those lines down there 😅